src/main/java/org/luksza/gerrit/rest/PostRetrigger.java

```diff
@@ -88,10 +104,36 @@ public Object apply(ChangeResource resource, Input input)
 
   private static final String STATUS_OPEN = "status:open";
 
+  private Optional<Header> getAuth(String jenkinsUrl) {
+    String userName = config.getJenkinsUserName(jenkinsUrl);
+    String token = config.getJenkinsToken(jenkinsUrl);
+    if (userName == null || token == null) {
+      return Optional.absent();
+    }
 
-  private Optional<String> createSession(HttpClient client, Project.NameKey project)
-      throws UnsupportedEncodingException, IOException, ClientProtocolException {
-    HttpPost search = prepareFormPost(urls.getSearchUrl(project));
+    String auth = userName + ":" + token;
+    byte[] encodedAuth = Base64.encodeBase64(auth.getBytes(Charset.forName("ISO-8859-1")));
```

Comment:
i would use utf-8 here instead of ISO-8859-1. They are compatible, but you won't get authentication issues when one use some strange characters in password